### PR TITLE
Automatically creates the token auth file if it isn't present

### PR DESCRIPTION
**Pull request by danielfm**
This PR improves UX after #418, hopefully avoiding issues like [this](https://github.com/kubernetes-incubator/kube-aws/issues/427#issuecomment-288551868).

### CODE REVIEW

File: core/controlplane/config/token_config.go [hunk 1]
```diff
@@ -152,6 +153,12 @@ func (r *EncryptedAuthTokensOnDisk) Compact() (*CompactAuthTokens, error) {
 
 func ReadOrEncryptAuthTokens(dirname string, encryptor CachedEncryptor) (*EncryptedAuthTokensOnDisk, error) {
 	authTokenPath := filepath.Join(dirname, "tokens.csv")
+
+	// Auto-creates the auth token file, useful for those coming from previous versions of kube-aws
+	if _, err := os.Stat(authTokenPath); os.IsNotExist(err) {
+		os.OpenFile(authTokenPath, os.O_RDONLY|os.O_CREATE, 0600)
```
Comment (mumoshu) [inline core/controlplane/config/token_config.go]: Almost nit but wouldn't it be safer to close the opened file afterwards?

**Comment by mumoshu** [comment]
1 nit but LGTM overall 👍 
Thanks for your efforts @danielfm!

File: core/controlplane/config/token_config.go (hunk 1 above)
Comment (danielfm) [inline core/controlplane/config/token_config.go]: Argh, forgot to close it. 😓 

@mumoshu just to let you know, feel free not to merge my PRs unless they are 100% according to your standards regarding code style, quality, and so on; just give me your feedback and I'll try to address them ASAP.

**Comment by mumoshu** [comment]
Thanks! I just tend to merge a PR if it looks 90% ok so that contributors
won't suffer from potential conflicts and rebasing while making a tiny fix.
Of course there's no problem for me to adjust to you.
2017年3月23日(木) 22:12 Daniel Martins <notifications@github.com>:

**Pull request merged**